environment.json

```diff
@@ -1,10 +1,12 @@
 {
   "DEV": {
     "baseUrl": "https://api.dev.p4pa.pagopa.it",
+    "baseUrlAuth": "https://api.dev.p4pa.pagopa.it/payhub-auth",
```

Comment:
don't do this!
otherwise we will complicate this file!
each MS should define its baseUrl, based on this 2 parameters, inside its `src/api/...` file

Reply:
modified in latest commit